templates/wordpress/1/rancher-compose.yml

```diff
@@ -0,0 +1,27 @@
+.catalog:
+  name: "Wordpress"
+  version: "v0.2"
+  description: "Blog tool, publishing platform and CMS"
+  minimum_rancher_version: v0.51.0
+  questions:
+    - variable: "public_port"
+      description: "Public port to access the Wordpress site"
+      label: "Public Port"
+      required: true
+      default: "80"
+      type: "int"
+    - variable: "mysql_root_password"
+      description: "Choose the MySQL root password"
+      label: "MySQL root password"
+      required: true
+      default: "wordpress"
+      type: "string"
+    - variable: "innodb_buffer_pool_size"
+      description: "Optionally tweak the InnoDB buffer pool size configuration setting"
+      label: "MySQL InnoDB Buffer Pool Size"
+      required: true
+      default: "8M"
+      type: "string"
```

Comment:
# Add service combo to select database service. You need to deploy it before.
    - variable: "wordpress_db_service"
      description: |
        Choose the database service to use. 
        It's really recommended a dedicated database service just for wordpress.
      label: "Database stack/service"
      default: "galera/galera-lb"
      required: true
      type: "service"